layouts/section/partnerships.html

```diff
@@ -68,59 +68,21 @@ <h2>{{ i18n "partnerships-team" | safeHTML }}</h2>
     <div class='row'>
       <div class='col-xs-12'>
         <h2>{{ i18n "what-were-working-on" }}</h2>
-        <p>{{ i18n "partnerships-partner-overview" }}
-
-          {{ $blog_url := "picking-our-projects" }}
-          {{ if eq $.Site.Language.Lang "fr" }}
-          {{ $blog_url = "choisir-nos-projets" }}
-          {{ end }}
-          <a href="{{with $.Site.GetPage $blog_url}}{{.Permalink}}{{ end }}">
-            {{ i18n "blog-post" }}.
-          </a>
-        </p>
       </div>
     </div>
   </div>
 
-  <!-- In-flight partnerships -->
   {{ $products := where .Site.Pages ".File.Dir" "==" "products/products/" }}
   <div class='container'>
-    <div class='row'>
-      <div class='col-xs-12'>
-        <h3>{{ i18n "partnerships-inflight" }}</h3>
-      </div>
+    <div class='row equal'>
+      {{ range $elem_index, $elem_val := $products }}
+      {{ if and (ne $elem_index 0) (eq (mod $elem_index 2) 0) }}
     </div>
     <div class='row equal'>
-      {{ $inflight := where .Site.Pages ".Params.status" "==" "in-flight" }}
-      {{ range $elem_index, $elem_val := $inflight | intersect $products }}
-        {{ if and (ne $elem_index 0) (eq (mod $elem_index 2) 0) }}
-          </div>
-          <div class='row equal'>
-        {{ end }}
-        <div class='col-md-6 col-xs-12'>
-          {{ partial "partnerships_product_item" $elem_val }}
-        </div>
       {{ end }}
-    </div>
-  </div>
-
-  <!-- Past partnerships -->
-  <div class='container'>
-    <div class='row'>
-      <div class='col-xs-12'>
-        <h3>{{ i18n "partnerships-past" }}</h3>
+      <div class='col-md-6 col-xs-12'>
+        {{ partial "partnerships_product_item" $elem_val }}
```

Comment:
This ends up causing an accessibility issue `Heading elements are not in a sequentially-descending order` as the partial starts at `h4` and the page last had an `h2` as the `h3` above was removed.